editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/rest/RestVerbPropertiesFilter.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.fusesource.ide.camel.editor.properties.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jface.viewers.IFilter;
+import org.fusesource.ide.camel.model.service.core.model.AbstractCamelModelElement;
+import org.fusesource.ide.camel.model.service.core.model.RestVerbElement;
+
+/**
+ * @author bfitzpat
+ */
+public class RestVerbPropertiesFilter implements IFilter {
+
+	/* (non-Javadoc)
+	 * @see org.eclipse.jface.viewers.IFilter#select(java.lang.Object)
+	 */
+	@Override
+	public boolean select(Object toTest) {
+		AbstractCamelModelElement ep = getSelectedObject(toTest);
+		List<String> restPropertyElements = new ArrayList<>();
```

Comment:
optional: this list can be a static List, it will avoid to recreate a new one on each call